docs/dev_guide/recommended_development_workflow.md

```diff
@@ -7,6 +7,14 @@ permalink: /docs/dev_guide/recommended_development_workflow
 
 # Venice Recommended Development Workflow
 
+## Create a Design Document
+
+If your change is relatively minor, you can skip this step. If you are adding new major feature, we suggest that 
```

Comment:
s/feature/functionality/

Reply:
Thanks for catching this in here as well.